carbon-design-system/carbon-addons-iot-react
# fix(header): tooltips for header action menu

Closes #3763 

**Summary**

- Add tooltip to action menus in Header

**Change List (commits, features, bugs, etc)**

- Replace anchor component to button, in order to display tooltip
- Override css styles
- Add unit tests
- Update snapshot

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3764--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-ui-shell-suiteheader-multi-workspace--admin-page)
- Hover over help and user icons
- Verify that tooltip has appeared

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
